### PR TITLE
[MRG] Maint/1.1

### PR DESCRIPTION
**Pull request by pavanramkumar**

### CODE REVIEW

**Comment by pavanramkumar** [comment]
@jasmainak i added the `.zenodo.json` config to this PR. not a huge fan of this approach since it's not tied to the release version and we must be careful that this file is updated at each subsequent release.

**Comment by titipata** [comment]
@pavanramkumar this looks good to me! thanks! I can merge it right away 👍

File: pyglmnet/__init__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Dev branch marker is: 'X.Y.devN' where N is an integer.
 #
 
-__version__ = '1.2.dev0'
+__version__ = '1.1.1'
```
Comment (jasmainak) [inline pyglmnet/__init__.py]: If we do this, we'll have to make a full release. I'd rather just do it on the maint branch.

**Comment by jasmainak** [comment]
@pavanramkumar pushed commit directly to master by cherry-pick.

**Pull request closed without merging**